app/boards/shields/helix/helix.conf

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+# Enables RGB functionality (Uncomment to enable.)
+# CONFIG_ZMK_RGB_UNDERGLOW=y
+# CONFIG_WS2812_STRIP=y
+
+# Enables OLED Function (TO DO. Do not uncomment.)
+# CONFIG_ZMK_DISPLAY=y
```

Comment:
This should probably be left out of this PR. It can be added if/when display support is added.
```suggestion
```

Reply:
Made adjustments to the five files as requested. If any syntax is off, please do inform me, and I'll adjust it when able.